Sources_v3/StreamChatUI/Common Views/ContainerStackView.swift

```diff
@@ -68,4 +68,12 @@ extension UIStackView {
         spacerView.setContentHuggingPriority(.defaultLow, for: .vertical)
         addArrangedSubview(spacerView)
     }
+    
+    func addSideMarginsToMatch(height: CGFloat) {
```

Comment:
What is the purpose of this?

Reply:
This thing was dynamically adding layoutMargins to stackView to layout content in the center. 
Kind of `spacerView` from both sides.
But I've already removed both spacer and this stuff.